docs/userGuide/components/searchbar.md

```diff
@@ -19,5 +22,6 @@
 Name | Type | Default | Description
 ---- | ---- | ------- | ------
 data | `Array` || The local data source for suggestions. Expected to be a primitive array.
+menu-align-right | `Boolean` | `false` | Whether the search bar's dropdown list will be right-aligned.
 on-hit | `Function` || A callback function when you click or hit return on an item.
 placeholder | `String` || 
```

Comment:
Empty string `''` for default. 

Also, should we add a short description as it looks out of place without one (all other rows have it)